### ISSUE TITLE
Test failures with 0.26.0

### ISSUE THREAD

**Issue by nmeum** [badaix/snapcast#961]
**Describe the bug**

While upgrading the Alpine Linux snapcast package to 0.26.0 I noticed some test failures of the tests provided by `snapcast_test`.  With version 0.25.0 all tests passed successfully.

**Steps to Reproduce**

1. Build snapcast with `-DBUILD_TESTS=ON` 
2. Run `./bin/snapcast_test`.

**Environment details**
- OS: Alpine Linux Edge
- Snapcast version: 0.26.0
- Installed from source using the Alpine [`APKBUILD`](https://gitlab.alpinelinux.org/alpine/aports/-/blob/953a4c1e411c913427c4bfb63c7367a4dde59dd7/community/snapcast/APKBUILD)

```
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
snapcast_test is a Catch v2.13.6 host application.
Run with -? for options

-------------------------------------------------------------------------------
Metadata
-------------------------------------------------------------------------------
/builds/alpine/aports/community/snapcast/src/snapcast-0.26.0/test/test_main.cpp:112
...............................................................................

/builds/alpine/aports/community/snapcast/src/snapcast-0.26.0/test/test_main.cpp:158: FAILED:
  REQUIRE( in_json == out_json )
with expansion:
  {"album":"Memories...Do Not Open","albumArtist":["The Chainsmokers"],
  "albumArtistSort":["Chainsmokers, The"],"artist":["The Chainsmokers &
  Coldplay"],"artistSort":["Chainsmokers, The & Coldplay"],"contentCreated":
  "2017-04-07","discNumber":1,"duration":247.0,"file":"The Chainsmokers -
  Memories...Do Not Open (2017)/05 - Something Just Like This.mp3","genre":
  ["Dance/Electronic"],"label":"Columbia/Disruptor Records",
  "musicbrainzAlbumArtistId":"91a81925-92f9-4fc9-b897-93cf01226282",
  "musicbrainzAlbumId":"a9ff33d7-c5a1-4e15-83f7-f669ff96c196",
  "musicbrainzArtistId":"91a81925-92f9-4fc9-b897-93cf01226282/cc197bad-dc9c-
  440d-a5b5-d52ba2e14234","musicbrainzReleaseTrackId":"8ccd52a8-de1d-48ce-acf9-
  b382a7296cfe","musicbrainzTrackId":"6fdd95d6-9837-4d95-91f4-b123d0696a2a",
  "originalDate":"2017","title":"Something Just Like This","trackNumber":5}
  ==
  {"album":"Memories...Do Not Open","albumArtist":["The Chainsmokers"],
  "albumArtistSort":["Chainsmokers, The"],"artist":["The Chainsmokers &
  Coldplay"],"artistSort":["Chainsmokers, The & Coldplay"],"contentCreated":
  "2017-04-07","discNumber":1,"duration":247.0,"genre":["Dance/Electronic"],
  "label":"Columbia/Disruptor Records","musicbrainzAlbumArtistId":"91a81925-
  92f9-4fc9-b897-93cf01226282","musicbrainzAlbumId":"a9ff33d7-c5a1-4e15-83f7-
  f669ff96c196","musicbrainzArtistId":"91a81925-92f9-4fc9-b897-93cf01226282/
  cc197bad-dc9c-440d-a5b5-d52ba2e14234","musicbrainzReleaseTrackId":"8ccd52a8-
  de1d-48ce-acf9-b382a7296cfe","musicbrainzTrackId":"6fdd95d6-9837-4d95-91f4-
  b123d0696a2a","originalDate":"2017","title":"Something Just Like This",
  "trackNumber":5}

{
    "canControl": false,
    "canGoNext": false,
    "canGoPrevious": false,
    "canPause": false,
    "canPlay": false,
    "canSeek": false,
    "loopStatus": "track",
    "playbackStatus": "playing",
    "position": 23.0,
    "shuffle": false,
    "volume": 42
}
-------------------------------------------------------------------------------
Properties
-------------------------------------------------------------------------------
/builds/alpine/aports/community/snapcast/src/snapcast-0.26.0/test/test_main.cpp:162
...............................................................................

/builds/alpine/aports/community/snapcast/src/snapcast-0.26.0/test/test_main.cpp:192: FAILED:
  REQUIRE( in_json == out_json )
with expansion:
  {"loopStatus":"track","playbackStatus":"playing","position":23.0,"shuffle":
  false,"volume":42}
  ==
  {"canControl":false,"canGoNext":false,"canGoPrevious":false,"canPause":false,
  "canPlay":false,"canSeek":false,"loopStatus":"track","playbackStatus":
  "playing","position":23.0,"shuffle":false,"volume":42}

Match: '[2021-06-04T07:20:47Z INFO  librespot_playback::player] <Tunnel> (310573 ms) loaded'
Match: '2021-06-04T07:20:47Z'
Match: ' '
Match: 'INFO'
Match: '  '
Match: 'librespot_playback::player'
Match: '<Tunnel> (310573 ms) loaded'
control:1
Category: control, Stream can not be controlled
===============================================================================
test cases:  6 |  4 passed | 2 failed
assertions: 68 | 66 passed | 2 failed
```

Full log: [alpine-edge-x86-snapcast.txt](https://github.com/badaix/snapcast/files/7771401/alpine-edge-x86_64-snapcast.txt)

**Comment by badaix** [comment]
This is not too severe as the test is erroneous, not the tested code.

**Comment by badaix** [comment]
Fixed: 411d98f5

**Comment by nmeum** [comment]
Thanks, tests pass now.
